appstore/middleware/filter_whitelist_middleware.py

```diff
@@ -18,9 +18,16 @@
 
 
 class AllowWhiteListedUserOnly(MiddlewareMixin):
+    def __init__(self, get_response=None):
+        if get_response is not None:
```

Comment:
This can be rewritten as 
```
self.get_response = get_response ? get_response : self._get_response
```
if you care about that kind of thing. No big deal though.

Reply:
Is Ternary too clever?